### PR TITLE
Move external toolchain PATH addition for the SDK

### PR DESCRIPTION
**Pull request by kergoth**
To ensure this is set correctly when using MULTILIBS, we need to directly add it to each environment setup script, so move it from sdk-env-external-toolchain to meta-toolchain, and also simplify and adjust how the value is set to base on our current external toolchain configuration, rather than constructing it from scratch.

JIRA: SB-16573

Signed-off-by: Christopher Larson <chris_larson@mentor.com>

### CODE REVIEW

**Comment by kergoth** [comment]
Confirmed that I can now build a hello world app for aarch64 with that env setup and arm with that multilib env setup, and that the resulting binaries are the correct arch.

File: core/recipes-core/meta/meta-environment.bbappend [hunk 1]
```diff
@@ -0,0 +1,26 @@
+EXTERNAL_TARGET_SYS ?= "${TARGET_ARCH}-oe-${TARGET_OS}"
```
Comment (abelal) [inline core/recipes-core/meta/meta-environment.bbappend:1]: Are we using this anywhere now?

Comment (kergoth) [reply core/recipes-core/meta/meta-environment.bbappend:1]: Good point, I'll drop it. Thanks.

**Review by abelal — APPROVED** [review]
:+1:

**Pull request merged**